torch/_dynamo/variables/torch.py

```diff
@@ -196,21 +194,22 @@ def call_function(
                 return self._call_cross_entropy_loss(tx, args, kwargs, options)
             else:
                 unimplemented(f"construct nn.Module: {self.value.__name__}")
+        elif self.value in (torch.is_tensor, torch.overrides.is_tensor_like):
+            assert len(args) == 1
+            if isinstance(args[0], TensorVariable):
+                return ConstantVariable(True, **options)
+            else:
+                return ConstantVariable(False, **options)
```

Comment:
For `torch.overrides.is_tensor_like` I don't think this is exactly right, see:
https://pytorch.org/docs/stable/torch.overrides.html#torch.overrides.is_tensor_like

Some user defined classes could return True if they define `__torch_function__`.  It should be easy to check for that case though.